lib/vet360/models/telephone.rb

```diff
@@ -61,7 +82,8 @@ def self.build_from(body)
           extension: body['phone_number_ext'],
           id: body['telephone_id'],
           is_international: body['international_indicator'],
-          is_textable: body['text_message_capable_ind'],
```

Comment:
Seems like we should either be handling both of these or neither

Reply:
I'm in favor of processing neither of these fields until there is more meaningful information we've got about them from Vet360.